### PR TITLE
871811 - Added leaf item content for tree map control.

### PR DESCRIPTION
**Pull request by Karthickmani97**
### Description

Added leaf item content for tree map control.

### CODE REVIEW

File: MAUI/TreeMap/leaf-item-settings.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+        set;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize leaf item appearance using style
+
+The leaf item appearance customization can be achieved by using the `TextStyle` property of `LeafItemSettings` in `SfTreeMap.` It is used to customize the `TextColor,` `FontSize,` `FontFamily,` and `FontAttributes` of tree map item text.
```
Comment (M-A32) [inline MAUI/TreeMap/leaf-item-settings.md:470]: The leaf item appearance customization can be achieved by using the `TextStyle` property of `LeafItemSettings` in `SfTreeMap.` It is used to customize the `TextColor,` `FontSize,` `FontFamily,` and `FontAttributes` of the treemap item text.

File: MAUI/TreeMap/selection-and-highlights.md [hunk 2]
```diff
@@ -287,7 +287,7 @@ public class PopulationViewModel
 
 ## Programatic item selection
 
-You can programmatically select the tree map items by utilizing the `SelectedItems` property in the `SfTreeMap`. This property holds a collection of tree map items representing the currently selected ones. In `Single` selection mode, only the first item in the SelectedItems collection will be selected. In `Multiple` selection mode, all items in the `SelectedItems` collection will be selected. When `SelectionMode` is set to `None`, no items are selected.
+You can programmatically select the tree map items by utilizing the `SelectedItems` property in the `SfTreeMap.` This property holds a collection of tree map items representing the currently selected ones. In `Single` selection mode, only the first item in the SelectedItems collection will be selected. In `Multiple` selection mode, all items in the `SelectedItems` collection will be selected. When `SelectionMode` is set to `None`, no items are selected.
```
Comment (M-A32) [inline MAUI/TreeMap/selection-and-highlights.md:290]: You can programmatically select the tree map items by utilizing the `SelectedItems` property in the `SfTreeMap.` This property holds a collection of the treemap items representing the currently selected ones. In the `Single` selection mode, only the first item in the SelectedItems collection will be selected. In the `Multiple` selection mode, all items in the `SelectedItems` collection will be selected. When `SelectionMode` is set to `None`, no items are selected.

File: MAUI/TreeMap/selection-and-highlights.md [hunk 3]
```diff
@@ -558,4 +558,6 @@ public class PopulationViewModel
 }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+N> This property is applicable only when the `SelectionMode` is set to `Single` or `Multiple.`
```
Comment (M-A32) [inline MAUI/TreeMap/selection-and-highlights.md:563]: N> This property is applicable only when the `SelectionMode` is set to `Single` or `Multiple`.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/TreeMap/leaf-item-settings.md [hunk 4]
```diff
@@ -0,0 +1,566 @@
...
+title: Leaf Item Settings in .NET MAUI TreeMap | Syncfusion
+description: Learn about the leaf items within Syncfusion .NET MAUI TreeMap, exploring its fundamental features and functionalities for effective treemap visualization.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Leaf item settings in .NET MAUI TreeMap (SfTreeMap)
+
+A leaf item in the TreeMap represents a visualized data element. It does not contain child items but may have a parent item if levels are specified in the TreeMap control. It also allows you to customize the spacing between items, stroke color, stroke width, text style, and text format option in the tree map control.
```
[the remark this answers is not in the document]
Comment (Karthickmani97) [reply MAUI/TreeMap/leaf-item-settings.md]: Updated.

File: MAUI/TreeMap/leaf-item-settings.md (hunk 1 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/leaf-item-settings.md]: Updated.

File: MAUI/TreeMap/selection-and-highlights.md (hunk 2 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/selection-and-highlights.md]: Updated.

File: MAUI/TreeMap/selection-and-highlights.md (hunk 3 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/selection-and-highlights.md:563]: Updated.

**Review by karthikraja-arumugam — APPROVED** [review]

**Pull request merged**